contracts/staking/StakingConstants.sol

```diff
@@ -95,4 +102,17 @@ contract StakingConstants {
         uint256 bzrxRewards,
         uint256 stableCoinRewards
     );
+
+    event AddAltRewards(
```

Comment:
I think those should be AddSushiRewards

Reply:
nope, this is generic event, if in future we will need to migrate to smth else or add other rewards type we just change address

Reply:
agreed